docs/UserGuide.md

```diff
@@ -30,152 +34,130 @@ FABook is a **desktop app for managing contacts, optimized for a financial advis
-  e.g. in `create n/NAME`, `NAME` is a parameter which can be used as `create n/John Doe`.
+* Words in `UPPER_CASE` are the information supplied by you.
+  e.g `NAME` is a parameter in `create n/NAME`, where you can input `create n/John Doe`. 
+  <br>
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [a/HOME_ADDRESS]` can be used as `n/John Doe a/Blk 30 Geylang Street 29` or as `n/John Doe`.
+* Parameters in `[]` are optional.
+  e.g `HOME_ADDRESS` is an optional parameter in `create n/NAME [a/HOME_ADDRESS]`, where you can input `create n/John Doe a/Blk 30 Geylang Street 29` or just `create n/John Doe`.
+  <br>
 
-* Items with `…`​ after them can be used multiple times.<br>
-  e.g. `NAME…​` can be used as `Jon`, `Jon Jack` etc.
+* Items with `…`​ after them can be used multiple times.
+  e.g `NAME…​` can be used as `Jon`, `Jon Jack` etc.
+  <br>
 
 * `INDEX` represents the index of a specific person in the FABook
-  e.g. `delete INDEX` can be used as `delete 2`.
+  e.g You can use the command `delete INDEX` as `delete 2`.
+<br>
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/HP_NUMBER`, `p/HP_NUMBER n/NAME` is also acceptable.
+* Parameters can be in **any order**.
+  e.g You can input either `create n/NAME p/HP_NUMBER` or ` create p/HP_NUMBER n/NAME`.
+  <br>
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+* Only the last occurrence of a parameter that is expected once will be taken **if you specify it more than once.**
+  e.g Given `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  <br>
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`,`clear`, 'redo' and 'undo') will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`,`clear`, 'redo' and 'undo') will be ignored.
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+ </div>
 
 ### Viewing help : `help`
 
-Shows a message explaining how to access the help page.
+Shows a message explaining how you can access our user guide.
 
-![Help](images/helpMessage.png)
-
-Format: `help` or press `F1`
+Format: `help` or Press `F1`
 
 ![help](images/UserGuide/help.png)
 
+
 ### Creating a contact: `create`
 
-Creates a contact that is stored in the FABook and contains their contact information. Any contact information that is not available can be updated later.
+Creates a contact with their contact information in your FABook. Any contact information you don't have on hand can be updated later.
 
-Format: `create n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [d/DESCRIPTION] [nw/NETWORTH] [mt/TIME] [t/TAG]...`
+Format: `create n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [d/DESCRIPTION] [nw/NETWORTH] [mt/TIME...] [t/TAG]`
```

Comment:
Now I'm a bit confused about what `...` implies.

Reply:
its only for things that can take more than one input which is only meeting times what no?